x-pack/test/api_integration/apis/siem/events.ts

```diff
@@ -95,6 +95,34 @@ const eventsTests: KbnTestProvider = ({ getService }) => {
             expect(events.edges[0]!.node.host!.name).to.eql([HOST_NAME]);
           });
       });
+      it('Make sure that timestamp is returned in the Events query', () => {
+        return client
+          .query<GetEventsQuery.Query>({
+            query: eventsQuery,
+            variables: {
+              sourceId: 'default',
+              timerange: {
+                interval: '12h',
+                to: TO,
+                from: FROM,
+              },
+              pagination: {
+                limit: 2,
+                cursor: CURSOR_ID,
+                tiebreaker: '193',
+              },
+              sortField: {
+                sortFieldId: 'timestamp',
+                direction: Direction.desc,
+              },
+              defaultIndex: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
+            },
+          })
+          .then(resp => {
+            const events = resp.data.source.Events;
+            expect(events.edges[0]!.node.timestamp).to.eql('2019-02-19T20:42:29.965Z');
+          });
```

Comment:
Nice test! 👍 . I ran my changes with it and everything worked out.

Reply:
Thanks! I think there's plenty of opportunity to beef up these api integration tests especially now that we've removed most the resolvers tests. They would check against an entire mock response object, which would've caught this. Good to know going forward! :)